src/tls.c

```diff
@@ -860,12 +896,16 @@ static INLINE word16 TLSX_ToSemaphore(word16 type)
 
 /** Checks if a specific light (tls extension) is not set in the semaphore. */
 #define IS_OFF(semaphore, light) \
-    ((semaphore)[(light) / 8] ^  (byte) (0x01 << ((light) % 8)))
+    (!(((semaphore)[(light) / 8] &  (byte) (0x01 << ((light) % 8)))))
```

Comment:
Was the change from XOR to AND on purpose and if so can you add comment for why it was wrong? Thanks.